plugins/modules/iosxr_logging.py

```diff
@@ -1089,7 +1151,7 @@ def main():
         hostnameprefix=dict(type="str"),
         level=dict(
             type="str",
-            default="informational",
+            default="debugging",
```

Comment:
I guess this was changed based on what the default is set in the doc for this key? Shouldn't it be the other way round i.e., updating the doc based on what's there in argspec? Changing the default value of a key in argspec can have other side effects.

Reply:
yes, this should be based on the default value in doc key as which is aligned with cisco IOS-XR default value for severity argument.

Reply:
So what I meant was, since the default value of a key is fetched from argument spec and not the doc while being used for computing, it is quite possible that changing it to something else may cause unintended effects and also break existing plays that rely on the default behaviour of level to be `debugging`.

Reply:
Having said that, I do not know what the default logging level is in IOS-XR and whether it varies between different versions.

Reply:
IMO we should go for what's there in ansible iosxr_logging supported by  iosxr platform documentation. What we have right now in our argument spec is a programming mistake or a potential bug.